.vscode/launch.json

```diff
@@ -17,10 +17,10 @@
             "buildFlags": "-tags e2e",
             "env": {
                 "OSMOSIS_E2E": "True",
-                "OSMOSIS_E2E_SKIP_IBC": "false",
```

Comment:
Let's sstop modifying these in every PR, please. I suggest having these all set to true so that we can do a quick local debug with no upgrade and other long-running tests

Reply:
yes, it also made sense to me so I left these in this PR